.babelrc

```diff
@@ -1,20 +1,25 @@
 {
+    "sourceMaps": "inline",
```

Comment:
this is theoretically the best way to get the sourcemaps into webpack for riot-web